doc/user/hierarchy.xml

```diff
@@ -643,43 +672,66 @@ env.Program('prog', ['prog.c'])
 
       <para>
 
-      The &Import; call makes the <literal>env</literal> construction
-      environment available to the &SConscript; file,
-      after which the variable can be used to build
-      programs, libraries, etc.
+      The &Import; call makes the previously defined <varname>env</varname> 
+      variable available to the &SConscript; file.
+      Assuming <varname>env</varname> is a &consenv;,
+      after import it can be used to build programs, libraries, etc.
+      The use case of passing around a &consenv; is extremely common
+      in larger &scons; builds.
 
       </para>
 
       <para>
 
       Like the &Export; function,
-      the &Import; function can be used
+      the &Import; function can be called
       with multiple variable names:
 
       </para>
 
       <sconstruct>
 Import('env', 'debug')
-env = env.Clone(DEBUG = debug)
+env = env.Clone(DEBUG=debug)
 env.Program('prog', ['prog.c'])
       </sconstruct>
+      
+      <para>
+
+      In this example, we pull in the common &consenv;
+      <varname>env</varname>, and
+      use the value of the <varname>debug</varname>
+      variable to make a modified copy by passing
+      that to a &f-link-Clone; call.
+
+      </para>
 
       <para>
 
-      And the &Import; function will similarly
-      split a string along white-space
+      The &Import; function will (like &Export;)
+      split a string containing white-space
       into separate variable names:
 
       </para>
 
       <sconstruct>
 Import('env debug')
-env = env.Clone(DEBUG = debug)
+env = env.Clone(DEBUG=debug)
 env.Program('prog', ['prog.c'])
       </sconstruct>
 
       <para>
 
+      &Import; prefers a local definition to a global one,
+      so that if there is a global export of <varname>foo</varname>,
+      <emphasis>and</emphasis> the calling SConscript has
```

Comment:
...callling SConscript() ? not SConscript?  the function call rather than the file or the context of the current sconscript?

Or maybe an example?

Reply:
in this case it refers to the file - the one that called this one.